src/button/button.style.js

```diff
@@ -110,33 +141,34 @@ const ButtonStyles = styled.div`
     }
   }
 
+  .quarternary .anchor-child,
```

Comment:
my editor says that there is a typo here quaternary not quarternary. Can someone confirm and then I will fix it